docs/tutorials/publish-rug-project.md

```diff
@@ -48,11 +45,16 @@ any channel that the Atomist Bot has been invited to.
   <img src="../images/atomist-team.png" alt="List of project generators" class="ss-medium">
 </div>
 
-The `sfz` Slack team ID is `T489LV5QF`. Look for your team Slack ID and pass it into
-the `rug publish` command like this.
+In this example, the Slack team ID is "T489LV5QF".
```

Comment:
It's worth checking with @cdupuis, but I think the CLI now supports Slack team name instead of Slack team ID as a parameter to publish.